vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.cpp

```diff
@@ -0,0 +1,793 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <DynamicReader.h>
+#include <DynamicWriter.h>
+#include <IReactContext.h>
+#include <IReactRootView.h>
+#include <Modules/PaperUIManagerModule.h>
+#include <Modules\NativeUIManager.h>
+#include <Views/ViewManager.h>
+#include <XamlUtils.h>
+#include "ShadowNodeBase.h"
+#include "Unicode.h"
+#include "XamlUIService.h"
+
+namespace Microsoft::ReactNative {
+
+struct ViewAtIndex {
```

Comment:
final?

Reply:
@asklar@microsoft.co: I've seen you make this comment many times ;) Should we consider writing a clang rule?

---
In reply to: [508115201](https://github.com/microsoft/react-native-windows/pull/6268#discussion_r508115201) [](ancestors = 508115201)

Reply:
sounds like a good idea :)